examples/with-emotion-10/package.json

```diff
@@ -12,11 +12,12 @@
   "author": "Thomas Greco",
   "license": "ISC",
   "dependencies": {
-    "emotion": "^8.0.11",
-    "emotion-server": "^8.0.11",
+    "@emotion/core": "10.0.27",
```

Comment:
this package.json probably shouldn't have the same `"name"` as the one targeting v11

Reply:
Done.